docs/content/getting-started/getting-help.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Getting Help | Dagster
+description: Have questions about how to use Dagster? Hit a bug? Have a feature request? This page includes tips on what to do.
+---
+
+# Getting Help
+
+Have questions about how to use Dagster? Hit a bug? Have a feature request? The Dagster Labs team and the Dagster community make a best-effort attempt to respond, on a few different platforms.
+
+---
+
+## Searching for answers
+
+Search engines like Google generally do a good job at surfacing relevant Dagster docs, Github Discussions and Github Issues, so we recommend starting there.
+
+Unfortunately, ChatGPT is not currently a great resource for answering questions about Dagster, because the corpus it was trained on doesn’t include Dagster’s latest APIs.
+
+---
+
+## Bugs and feature requests
+
+We use [Github Issues](https://github.com/dagster-io/dagster/issues) to track all bugs and feature requests from the community, and we welcome issues submitted by users.
+
+If you find a Github issue that seems like it’s related to the issue you’re experiencing, but you're not sure, don’t be shy about posting on that Github issue to ask.  We can redirect it to a different issue if necessary.
+
+### Tips for filing issues
+
+- For bugs, include the minimum-sized working code snippet that reproduces your issue.
+- If your Github issue includes a code snippet, you can add syntax highlighting by specifying the language at the top of the block:
+
+  ```
+  ```python
+  from dagster import ...
+  ...
+  ```
```

Comment:
missing closing "```" - now it renders as:
<img width="309" alt="image" src="https://github.com/dagster-io/dagster/assets/4531914/bf59eb03-7e67-486d-9fd8-605ea7541096">

Reply:
I tried a few things but could not for the life of me figure out how to get the closing "```" in there. Do you know how to do this?

Reply:
did you try this? It renders correctly for me when I try it locally:

```suggestion
- If your Github issue includes a code snippet, you can add syntax highlighting by specifying the language at the top of the block:

      ```python
      from dagster import ...
      ...
      ```

```

There are escape characters in Markdown that would make this easier (ex: `~~~`) but the current parser unfortunately strips them out when the formatting script is run.

Reply:
Is the difference the indentation? I'll try that.

Reply:
Exactly that!

Reply:
worked!